app/assets/stylesheets/application.css.scss

```diff
@@ -15,8 +15,29 @@
  *= require foundation_and_overrides
  */
 
+.logo {
+  background-color: #d9b82d;
+  border-color: #111111;
+  margin-bottom: 0px;
+  margin-top: 10%;
+  text-align: center;
+  text-shadow: -1px 0 white, 0 1px white, 1px 0 white, 0 -1px white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.